### PR TITLE
Task/146 overdracht feedback

### PR DESCRIPTION
**Pull request by RubenEppink**

### CODE REVIEW

**Review by mjqnsen — APPROVED** [review]

File: wcag-tool/package.json [hunk 1]
```diff
@@ -1,9 +1,13 @@
 {
```
Comment (ilivss) [inline wcag-tool/package.json:1]: :ok_hand:

File: README.md [hunk 2]
```diff
@@ -6,86 +6,130 @@ Execute the following steps in the given order to setup the development environm
...
 1. Navigate to `\wcag-tool`
 2. Run `npm run setup`
-3. Run `npm run dev`
+2. Run `npm run seed:dev`
+3. Run `npm run start`
 4. Open a chrome browser and browse to `chrome://extensions/`
 5. Make sure developer mode is checked. 
 6. Click on `Load unpacked`
 7. Select the following folder `\wcag-tool\chrome-extension\dist` 
+7. Make sure you database instance is running if you're using a local Mongo instance.
```
Comment (RubenEppink) [inline README.md:78]:
```suggestion
7. Make sure your database instance is running if you're using a local Mongo instance.
```

File: wcag-tool/chrome-extension/src/services/apiService.js [hunk 3]
```diff
@@ -1,8 +1,9 @@
 import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react'
+import config from '../../config'
 
 export const api = createApi({
   reducerPath: 'api',
-  baseQuery: fetchBaseQuery({ baseUrl: 'http://localhost:5000/v1/' }),
+  baseQuery: fetchBaseQuery({ baseUrl: config.SERVER_URL }),
```
Comment (ilivss) [inline wcag-tool/chrome-extension/src/services/apiService.js:6]: :ok_hand:

File: wcag-tool/chrome-extension/tests/unit/utils.index.test.js [hunk 4]
```diff
@@ -18,7 +18,7 @@ describe('unit truncateStringAndCapitalize', () => {
   })
   it('should not work with null', () => {
     expect(() => truncateStringAndCapitalize(0, null)).toThrow(
-      "Cannot read properties of null (reading 'charAt')"
+      "Cannot read property 'charAt' of null"
```
Comment (RubenEppink) [inline wcag-tool/chrome-extension/tests/unit/utils.index.test.js:21]:
```suggestion
      "Cannot read properties of null (reading 'charAt')"
```

File: README.md [hunk 5]
```diff
@@ -6,86 +6,130 @@ Execute the following steps in the given order to setup the development environm
...
+> Installs all dependencies for the backend and frontend. 
+
+> **npm run start**
+> Starts up the backend with node and builds the frontend into the dist folder that needs to be deployed to the chrome store.
+
+> **npm run test:unit**
+> Run all unit tests for both front- and backend. 
+
+> **npm run test:integration**
+> Run all integration tests for both front- and backend.
```
Comment (mjqnsen) [inline README.md:99]:
```suggestion
> Run all integration tests for backend.  
```

File: README.md [hunk 6]
```diff
@@ -6,86 +6,130 @@ Execute the following steps in the given order to setup the development environm
...
+
+> **npm run start**
+> Starts up the backend with node and builds the frontend into the dist folder that needs to be deployed to the chrome store.
+
+> **npm run test:unit**
+> Run all unit tests for both front- and backend. 
+
+> **npm run test:integration**
+> Run all integration tests for both front- and backend.  
+
```
Comment (mjqnsen) [inline README.md:100]:
```suggestion

> **npm run test:e2e**
> Run all e2e tests.
```

**Review by mjqnsen — CHANGES_REQUESTED** [review]

File: README.md [hunk 7]
```diff
@@ -6,86 +6,132 @@ Execute the following steps in the given order to setup the development environm
...
+> **npm run test:unit**
+> Run all unit tests for both front- and backend. 
+
+> **npm run test:integration**
+> Run all integration tests for backend.  
+
+> **npm run test:e2e**
+> Run all e2e tests.
+> **npm run lint**
```
Comment (mjqnsen) [inline README.md:103]:
```suggestion

> **npm run lint**
```

**Review by mjqnsen — APPROVED** [review]

**Review by ilivss — APPROVED** [review]
👌🏽

**Pull request merged**